articles/010_why_ros2.md

```diff
@@ -17,7 +19,9 @@ categories: Overview
 {{ page.abstract }}
 </div>
 
-Original Author: {{ page.author }}
+Original Author: {{ page.author }}  
+Written: {{ page.written }}  
```

Comment:
I prefer "Date Written" here, but it is admittedly subjective.  I'll leave it up to you whether to make the change.

Reply:
sure that's fine, added in 1436a427a3c7927edd127f20c312aa0485194217